packages/create-block/lib/output.js

```diff
@@ -19,7 +19,12 @@ const writeOutputTemplate = async ( inputFile, outputFile, view ) => {
 		outputFile.replace( /\$slug/g, view.slug )
 	);
 	await makeDir( dirname( outputFilePath ) );
-	writeFile( outputFilePath, render( inputFile, view ) );
+	// If the rendered template is empty, don't write it. This is how we can conditionally add template files.
+	const renderedFile = render( inputFile, view ).trim();
+	if ( renderedFile.length ) {
+		// We need to re-render so that the whitespace at the end of files are maintained
+		writeFile( outputFilePath, render( inputFile, view ) );
+	}
```

Comment:
This would work better:
```suggestion
	const renderedFile = render( inputFile, view );
	if ( renderedFile.trim().length ) {
		// We need to re-render so that the whitespace at the end of files are maintained
		writeFile( outputFilePath, renderedFile );
	}
```

Reply:
I did something similar here and when the files were output ( specifically the .scss ) they were triggering linting errors due to missing trailing spaces.

Reply:
The trick is that I propose to run trimming only when evaluating the condition in the `if` statement. The content of the rendered file should remain unchanged when saving to the file.

Reply:
ah! I see! That _IS_ a much better way. I'll update the PR.